Makefile

```diff
@@ -140,6 +165,3 @@ dockerenvbuild: writeversion
 dockerenvdist: writeversion
 	docker run --env-file ./env.list -v $$PWD/dist:/kolibridist learningequality/kolibri:$$(cat kolibri/VERSION)
 
-kolibripippex:
```

Comment:
This seemed unused, couldn't find any mentions in the codebase

Reply:
I originally put it there to get the kolibripip.pex to use in `install_cexts.py`. But since we have the kolibripip.pex in the repository already, I agree that removing it makes our `Makefile` more clean! 👍